src/main/scala/cromwell/Main.scala

```diff
@@ -68,7 +67,10 @@ object Main extends App {
         case _ => throw new RuntimeException("Expecting a JSON object")
       }
 
-      val workflowManagerSystem = new WorkflowManagerSystem {}
+      lazy val realDataAccess = new RealDataAccess
```

Comment:
I know you mentioned renaming elsewhere but I'm putting in a vote against `Real` and `Dummy` :) If `Real` is MySql specific (is it?) perhaps `MySqlDataAccess`, otherwise perhaps `DbBasedDataAccess`. I care a lot less what the test one is called.